### DISCUSSION TITLE
Space Weather / Aurora Predictions / Radio propagation page

### DISCUSSION THREAD

**Discussion by JonFear** [Daveiano/weewx-wdc#137] [Ideas]
Sorry Dave,

Not sure how much mileage is in this proposal.

As we are heading for solar maxima over the next couple of years, having a simple page showing Kp level and perhaps Aurora predictions might be an idea. Space weather and Radio propagation is very much linked with this.

References: 
https://www.spaceweatherlive.com/en.html
https://www.hamqsl.com/solar.html#addwebsite

Thoughts?

Jon

**Comment by Daveiano** [comment]
You don't need to excuse for new ideas!

Solar maxima is predicted for 2025, right? 

> ... having a simple page showing Kp level and perhaps Aurora predictions might be an idea. Space weather and Radio propagation is very much linked with this.

Yes, I absolutely like this idea! Since I am doing some Astro-Photography in my spare free time, I am also into these topics. I quickly looked over the two example sites and I could not find any "Add to website" kind of embed things on the first one, did I miss something? The second page provides copy&paste iFrame codes, which could be easily added to the skin (I could provide some guidance on that).

Do you have any other examples for easy-to-integrate charts, etc from third-party sites? Or do you know about any existing weewx extensions providing such data? 

Also, wondering if this could also be added to the existing celestial page?

**Reply by JonFear** [reply]
Hi Dave

Any thing Space Weather, Radio propagation, Astro Photography related items will go down well with me as well! I will take a look during the weekend and get together a list of sites and check whether we can legally harvest data from them...

BW

Jon
